modules/core/misc/python/shadow_umat.hpp

```diff
@@ -0,0 +1,60 @@
+#ifdef HAVE_OPENCV_CORE
+
+#include "opencv2/core/mat.hpp"
```

Comment:
> #ifdef HAVE_OPENCV_CORE

Check is not necessary.

> #include "opencv2/core/mat.hpp"

This file is not intended for processing by C++ compiler. Only bindings parser should handle this file.
Perhaps you can leave short comment message here about that.

Reply:
Done!